sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -27,6 +28,23 @@
  */
 public abstract class CouchTestBase {
 
+
+    static {
+        try {
+            if (Misc.isRunningOnAndroid()) {
```

Comment:
Also any tests which don't inherit from `CouchTestBase` won't get their config, is this a concern?